### PR TITLE
Rename Kingdom completed

### PR DESCRIPTION
**Pull request by AggieK**

### CODE REVIEW

**Review by szutsj — APPROVED** [review]

File: src/main/resources/application.properties.example [hunk 1]
```diff
@@ -1,7 +1,7 @@
 spring.datasource.url=${TGB_APP_DB_URL}
 spring.datasource.username=${TGB_APP_DB_USERNAME}
 spring.datasource.password=${TGB_APP_DB_PASSWORD}
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=update
```
Comment (balazsbarni) [inline src/main/resources/application.properties.example:4]: ezt miert csereltuk ki? az exampleban boven jo a `create-drop`

**Review by balazsbarni — CHANGES_REQUESTED** [review]
S ha megy nyomsz egy csodabillentyu kombinaciot, hogy az indentationt csinalja meg szuper lesz. Amugy szep munka

File: src/main/resources/application.properties.example (hunk 1 above)
Comment (AggieK) [reply src/main/resources/application.properties.example:4]: Hi, indentation corrected. We made some print screens for Judit's presentation, that is why the extra line brakes. Deleted & corrected. IntelliJ deletes the Application.properties file when master pulled - magic. Example file set to update DB due easy copy-paste routine.

Comment (balazsbarni) [reply src/main/resources/application.properties.example:4]: ok

Comment (balazsbarni) [reply src/main/resources/application.properties.example:4]: in the readme then this should be mentioned

**Review by balazsbarni — APPROVED** [review]

**Pull request merged**